### PR TITLE
Brand name lookup

### PR DESCRIPTION
**Pull request by viacoffee**
Lookup the complete brand name based on the prefix that we split out of product name.

### CODE REVIEW

File: lib/bill_hicks/brand_converter.rb [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+  class BrandConverter
+    def self.convert(value)
+      if brand = @brands.detect {|brand| brand[:prefix] == value.split[0] }
+        brand[:company]
+      else
+        value.split[0]
+      end
+    end
+
+    @brands = [
```
Comment (Oshuma) [inline lib/bill_hicks/brand_converter.rb]: Shouldn't this be a class var (`@@brands`)?

Comment (jwilsjustin) [reply lib/bill_hicks/brand_converter.rb]: Or better yet, a class variable memoized in a class method.

Comment (Oshuma) [reply lib/bill_hicks/brand_converter.rb]: Even better. :+1:

Comment (viacoffee) [reply lib/bill_hicks/brand_converter.rb]: :+1:

**Pull request merged**